### PR TITLE
Extend ACD documentation

### PR DESCRIPTION
**Pull request by maximiliankolb**
* Fix path for ACD Ansible playbooks
* Add git support to Ansible playbook
* Extend ACD user roles
* Git branch commit tag no longer optional

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Friendly reminder @melcorr and anyone else who would be so kind to review this/give feedback.

**Comment by melcorr** [comment]
On my list @maximiliankolb :) 
Sorry for delay.

File: guides/common/modules/proc_adding_an_ansible_playbook.adoc [hunk 1]
```diff
@@ -8,13 +8,22 @@ You can add xref:{context}_ansible_playbooks[Ansible playbooks] used for applica
 . Click *New Ansible Playbook*.
 . In the *Name* field, enter the name of new Ansible playbook.
 . In the *Description* field, enter an arbitrary description.
-. In the *Path* field, enter the location of the Ansible playbook on your {ProjectServer}.
+. In the *SCM Type* drop down menu, select either _directory_ if the Ansible playbook is located on your {ProjectServer} or _git_ to reference a remote git repository.
+** If you select _directory_, enter the location of the Ansible playbook on your {ProjectServer} in the *Directory Path* field.
 +
-To avoid SELinux issues, add the Ansible playbook to the `/etc/foreman/plugins/foreman_acd/ansible-playbooks/` directory.
+To avoid SELinux issues, add the Ansible playbook to the `/var/lib/foreman/foreman_acd/ansible-playbooks/` directory.
+** If you select _git_, enter the remote location in the *Git Url* field and a git branch, commit, or tag in the *Git Branch/Commit/Tag* field.
```
Comment (melcorr) [inline guides/common/modules/proc_adding_an_ansible_playbook.adoc:15]:
```suggestion
** If you select *git*, enter the remote location in the *Git Url* field and a git branch, commit, or tag in the *Git Branch/Commit/Tag* field.
```

File: guides/common/modules/proc_adding_an_ansible_playbook.adoc [hunk 2]
```diff
@@ -8,13 +8,22 @@ You can add xref:{context}_ansible_playbooks[Ansible playbooks] used for applica
...
++
+Click the *Sync Repository* button to fetch the remote git repository.
```
Comment (melcorr) [inline guides/common/modules/proc_adding_an_ansible_playbook.adoc:17]:
```suggestion
Click *Sync Repository* to fetch the remote git repository.
```

IBM Style

**Review by melcorr — CHANGES_REQUESTED** [review]
tiny nits, nothing urgent or blocking.

File: guides/common/modules/con_acd_architecture.adoc [hunk 3]
```diff
@@ -96,7 +96,17 @@ Refer to the https://docs.ansible.com/ansible/latest/user_guide/playbooks_variab
 The ACD plugin requires the specific {ProjectName} permissions.
 Depending on your environment, there might be two different groups of users in regard to _managing_ an application and _deploying_ an application:
 
+Users can interact with three different resources for ACD:
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/con_acd_architecture.adoc:99]: Fixed.

File: guides/common/modules/proc_adding_an_ansible_playbook.adoc [hunk 4]
```diff
@@ -8,13 +8,22 @@ You can add xref:{context}_ansible_playbooks[Ansible playbooks] used for applica
 . Click *New Ansible Playbook*.
 . In the *Name* field, enter the name of new Ansible playbook.
 . In the *Description* field, enter an arbitrary description.
-. In the *Path* field, enter the location of the Ansible playbook on your {ProjectServer}.
+. In the *SCM Type* drop down menu, select either _directory_ if the Ansible playbook is located on your {ProjectServer} or _git_ to reference a remote git repository.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_adding_an_ansible_playbook.adoc:11]: Applied.

File: guides/common/modules/proc_adding_an_ansible_playbook.adoc [hunk 5]
```diff
@@ -8,13 +8,22 @@ You can add xref:{context}_ansible_playbooks[Ansible playbooks] used for applica
...
+** If you select _directory_, enter the location of the Ansible playbook on your {ProjectServer} in the *Directory Path* field.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_adding_an_ansible_playbook.adoc:12]: I think they're neat. However, removed one `*`.

File: guides/common/modules/proc_adding_an_ansible_playbook.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_adding_an_ansible_playbook.adoc:15]: Applied.

File: guides/common/modules/proc_adding_an_ansible_playbook.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_adding_an_ansible_playbook.adoc:17]: Old habits, you know...
Fixed!

**Comment by maximiliankolb** [comment]
Thanks for review! Resolved all issues and rebased to master.

**Review by melcorr — APPROVED** [review]
LGTM and thank you for applying the feedback!

**Pull request merged**